src/types.js

```diff
@@ -18,9 +18,7 @@ export type AlignGrid = AlignItem | 'stretch'
 
 export type Justify = 'left' | 'center' | 'right' | void
 
-export type Margin = 'none' | 'all' | 'horizontal' | 'vertical' | void
```

Comment:
Now everything uses `IndividualSides` instead. "horizontal", now is the same than `"left right"`, `"left left right right"`, `"horizontal"`, `"horizontal right"` and we cannot account for all permutations. Because of this `Margin` type is no longer used.

I think this highlights the increased consistency of the API since now all margins and padding are treated the same way